asset/asset.go

```diff
@@ -28,6 +28,11 @@ type SerializedKey [33]byte
 // 32-byte form.
 type SchnorrKey [32]byte
 
+// GroupKeyReveal is a type for representing the data used to derive a group key
+// for an asset. Specifically, internal_key || tapscript_root.
+// TODO(jhb): Implement constructor for this field that includes length checks
+type GroupKeyReveal [GroupKeyRevealLen]byte
```

Comment:
Would it make more sense to make this a struct 
```go
type GroupKeyReveal {
    Key [32]byte
    Root [32]byte
}
```
that we could deserialize directly into?